cmake/Helpers.cmake

```diff
@@ -131,17 +131,30 @@ macro(print_info msg)
   message("${Blue}${msg}${ResetColor}")
 endmacro()
 
-# Helper to print the configuration of a 2-valued or 3-valued option 'var'
-# with prefix 'str'.
+# Helper to print the configuration of a 2-valued or 3-valued option 'var' with
+# prefix 'str'. Optionally takes a `FOUND_SYSTEM` argument that is used to
+# indicate when a given dependency is built as part of the cvc5 build.
 function(print_config str var)
+  set(options)
+  set(oneValueArgs FOUND_SYSTEM)
+  set(multiValueArgs)
+  cmake_parse_arguments(ARGS "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+
   if("${var}" STREQUAL "ON")
     set(OPT_VAL_STR "on")
   elseif("${var}" STREQUAL "OFF" OR "${var}" STREQUAL "IGNORE")
     set(OPT_VAL_STR "off")
   else()
     set(OPT_VAL_STR "${var}")
   endif()
-  message("${Blue}${str}: ${Green}${OPT_VAL_STR}${ResetColor}")
+
+  if("${ARGS_FOUND_SYSTEM}" STREQUAL "FALSE")
+    set(OPT_FOUND_SYSTEM_STR " (building)")
```

Comment:
Would it make sense to maybe switch this and print `(system)` when it's found in the system? I guess it depends what we consider the default case.

Reply:
I'd suggest to do both: either `(system)` or `(building)` (or maybe `(local)`?)

Reply:
Yeah, I was thinking about having both, too. I don't have a strong preference either way but maybe specifying both is a good idea because it allows us to distinguish between that information being present or not.